src/Promitor.Core.Scraping/Configuration/Serialization/v1/Core/AzureMetadataDeserializer.cs

```diff
@@ -20,18 +19,19 @@ public AzureMetadataDeserializer(ILogger<AzureMetadataDeserializer> logger) : ba
             Map(metadata => metadata.ResourceGroupName)
                 .IsRequired();
             Map(metadata => metadata.Cloud)
-                .WithDefault(AzureEnvironment.AzureGlobalCloud)
+                .WithDefault(AzureCloud.Global)
                 .MapUsing(DetermineAzureCloud);
         }
 
+        // TODO: validate cloud configuration in a SDK-agnostic way
```

Comment:
Can you open an issue and link to it here please?

Reply:
I used  azureCloud.DetermineMetricsClientAudience() for validation, see comment above